### ISSUE TITLE
rtree gets confused when a remote is referenced both with alias and without

### ISSUE THREAD

**Issue by majewsky** [majewsky/gofu#2]
```bash
$ cd /
$ cg gh:lib/pq
!! /x/src/github.com/lib/pq already exists (if there is a repo there, try `rtree index`)
$ pwd
/
$ cg https://github.com/lib/pq
$ pwd  
/x/src/github.com/lib/pq
```

Plan:
- [ ] always write repositories into index with their fully expanded remote URL
- [ ] when displaying remote URL, apply available aliases to shorten the URL

**Comment by talal** [comment]
bump.

Do you have a fix for this? 
I am asking in case you have fixed already but haven't gotten around to committing it yet 😄

**Comment by majewsky** [comment]
No, I was too lazy. ^^ If you want to implement the fix that I've described up there, I'd happily accept a PR.

**Comment by talal** [comment]
I'll try and see if I can come up with something.
